app/src/main/AndroidManifest.xml

```diff
@@ -25,17 +25,15 @@
         <activity
             android:name=".frontend.activity.ClientRegistrationActivity"
             android:exported="false"
-            android:parentActivityName=".frontend.activity.LoginActivity"
-            android:theme="@style/Theme.AppCompat.NoActionBar">
```

Comment:
We might want to leave the no action bar in

Reply:
Can you explain why? We are on Client registration activity so wouldn't we want to go back to login page if we don't want register?